superset/db_engine_specs/mysql.py

```diff
@@ -93,6 +104,21 @@ class MySQLEngineSpec(BaseEngineSpec):
 
     type_code_map: Dict[int, str] = {}  # loaded from get_datatype only if needed
 
+    custom_errors = {
+        INVALID_ACCESS_REGEX: (
+            __('Either the username "%(username)s" or the password is incorrect.'),
+            SupersetErrorType.TEST_CONNECTION_ACCESS_DENIED_ERROR,
+        ),
+        INVALID_HOSTNAME_REGEX: (
+            __('Unknown MySQL server host "%(hostname)s"'),
+            SupersetErrorType.TEST_CONNECTION_INVALID_HOSTNAME_ERROR,
+        ),
+        CONNECTION_HOST_DOWN_REGEX: (
```

Comment:
Same here, let's call this just `HOST_DOWN_REGEX`.

Reply:
Ah, I just saw that this already existed, sorry.

But I think we should call this `TEST_CONNECTION_HOST_DOWN_REGEX`, to match the error type.

Reply:
Let's use that pattern as we add regexes.

Reply:
this makes sense